starlark/crashd_config.go

```diff
@@ -63,6 +67,16 @@ func crashdConfigFn(thread *starlark.Thread, b *starlark.Builtin, args starlark.
 		return starlark.None, fmt.Errorf("%s: %s", identifiers.crashdCfg, err)
 	}
 
+	if useSSHAgent {
+		if agent := ssh.GetAgent(); agent == nil {
+			logrus.Debug("starting ssh-agent")
+			_, err := ssh.StartAgent()
```

Comment:
Oh ok I see, you starting it here.

Reply:
So, user of ssh (yourself) should be protected when calling `ssh.GetAgent`. Because, if a nil is returned (without starting), then panic.

Reply:
Added extra check for `nil` agent ref.

Reply:
If I make the change to enable the `GetAgent()` to start the agent, if not found, then we cannot directly use the GetAgent() call here.
https://github.com/vmware-tanzu/crash-diagnostics/pull/165/files#diff-d77ccf1d8ab167af985be6f0a10f043aR47-R50